docker-compose.yml.BACKUP

```diff
@@ -0,0 +1,122 @@
+version: "3.7"
```

Comment:
Can this file be removed?

Reply:
Thanks - I have missed that one. File removed from git cache.